sound/soc/sof/ops.h

```diff
@@ -234,7 +234,7 @@ static inline int snd_sof_dsp_is_ready(struct snd_sof_dev *sdev)
 	if (sdev->ops->is_ready)
 		return sdev->ops->is_ready(sdev);
 
-	return 0;
+	return 1;
```

Comment:
@plbossart Because this is an "is_<condition>" function, which is used in ipc.c as `if (snd_sof_dsp_is_ready(...))` and platform implementations (e.g. byt_is_ready()) return 1 for "yes, ready" and 0 for "busy." So, the semantics seem to be 1 for ready and 0 for busy, and we want to return "ready" by default, IIUC?